src/Tizen.NUI/src/internal/Xaml/ApplyPropertiesVisitor.cs

```diff
@@ -211,7 +212,7 @@ public void Visit(ElementNode node, INode parentNode)
                 MethodInfo addMethod;
                 if (xpe == null && (addMethod = collection.GetType().GetRuntimeMethods().First(mi => mi.Name == "Add" && mi.GetParameters().Length == 1)) != null)
                 {
-                    addMethod.Invoke(collection, new[] { Values[node] });
+                    addMethod.Invoke(collection, new[] { value });
```

Comment:
please resolve the VS warning below.

**src/Tizen.NUI/src/internal/Xaml/ApplyPropertiesVisitor.cs#L646
Modify 'GetConvertedValue' to catch a more specific allowed exception type, or rethrow the exception.**